File: tournaments/swapCase/swapCase.js
function swapCase(text) {
    const n = text.length;
    const result = [];

    for (const item of text) {
        if ('A' <= item && 'Z' >= item) {
            result.push(String.fromCharCode(item.charCodeAt() - 'A'.charCodeAt() + 'a'.charCodeAt()));
        } else {
            result.push(String.fromCharCode(item.charCodeAt() - 'a'.charCodeAt() + 'A'.charCodeAt()));
        }
    }

    return result.join('');
}

File: tournaments/swapCase/swapCase.cpp
std::string swapCase(std::string text) {

  std::string answer = "";

  for (int i = 0; i < text.length(); i++) {
    if (text[i] == std::toupper(text[i])) {
      answer += std::tolower(text[i]);
    }
    else {
      answer += std::toupper(text[i]);
    }
  }
  return answer;
}
